WcaOnRails/app/views/registrations/edit_registrations.html.erb

```diff
@@ -1,6 +1,7 @@
 <% provide(:title, I18n.t('registrations.list.title', comp: @competition.name)) %>
 
 <%= render layout: 'nav' do %>
+  <%= link_to icon("plus", "Show events for each registration"), competition_edit_registrations_path(@competition, show_events: "true"), class: "btn btn-primary" %>
```

Comment:
I think this should change to a "- Do not show events for each registration" whenever `show_events=true` is a url parameter.